src/media/media.controller.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  Body,
+  Controller,
+  Post,
+  UploadedFile,
+  UseInterceptors,
+} from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
+import type { MediaUploadDTO } from './dto/mediaUpload.dto';
+import type { uploadFIle as IuploadFile } from './interface/uplaodFile.interface';
+import { MediaService } from './media.service';
+
+@Controller('media')
+export class MediaController {
+  constructor(private mediaService: MediaService) {}
+  @Post()
+  @UseInterceptors(FileInterceptor('file'))
+  async uploadFile(
+    @Body() mediaUploadDto: MediaUploadDTO,
```

Comment:
I've sent a request using Postman, where I only included the image file, and did not include the entity_id and entity_option, and the request was sent successfully and a response {ack: true} was received.

Shouldn't this be regarded as a bad request?

Reply:
yes, there was a bug in code and the validations weren't running successfully. All validations were fixed